cloud-functions/src/index.ts

```
@@ -337,3 +338,46 @@ export const vercelLogDrain = functions.https.onRequest(async (req, res) => {
+				await networkDocRef.set(
+					{
+						token_price: {
+							value: latestTokenPrice,
+							last_fetched_at: admin.firestore.Timestamp.now()
+						}
+					},
+					{ merge: true }
+				);
+			} catch (error) {
+				console.error(`Error updating token price for ${networkName}:`, error);
+				continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)